src/api.ts

```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+
+import { log } from './utils/log';
+
+export function createAPI(): void {
+  const server = http.createServer((req, res) => {
```

Comment:
do we know what the overhead of this server will be?

Reply:
there is not much overhead for a simple open connection,
It's not exposed to the public it's just for internal health check, pinged every 10-30s